playbooks/roles/jenkins_master/handlers/main.yml

```diff
@@ -3,13 +3,22 @@
   service:
     name: jenkins
     state: restarted
+  tags:
```

Comment:
What's the purpose of adding tags to a handler? My understanding is that the handler will be run whenever a tasks contains `notify`. Do the tags change this behavior somehow?

Reply:
This has been introduced when I pulled last upstream changes here. Maybe is to be able to perform small changes, that aren't hooked up with the handler.